changes/ce/fix-14793.en.md

```diff
@@ -0,0 +1,5 @@
+Add trace log for `protocol_error`.
+
+Previously when client send invalid or unexpected packets to cause a `protocol_error` in EMQX, the detailed reason is not traced.
+For example, if a client send two `CONNECT` packets, for the second one, EMQX may log `socket_force_closed` with `protocol_error`, but hard to tell why.
+Now EMQX will log `unexpected_connect_packet` with `conn_satet=connected` before `socket_force_closed`.
```

Comment:
Curiously enough, same typo also sneaked in here also.

Reply:
copy-paste ;) fixed.